src/components/RefinementList/RefinementList.tsx

```diff
@@ -42,6 +47,12 @@ export type RefinementListComponentCSSClasses = RefinementListRequired & {
     Pick<HierarchicalMenuComponentCSSClasses, 'childList' | 'parentItem'>
   >;
 
+// Templates types
```

Comment:
Not sure these comments are useful.

Reply:
True, removed in: https://github.com/algolia/instantsearch.js/pull/4785/commits/97c8c9a82afbb7a7462a57fee4a25a44580ff14b